### ISSUE TITLE
DSS 5.1.6 Crash

### ISSUE THREAD

**Issue by astrobob56** [deepskystacker/DSS#207]
**Check the Wiki**
Before you report any issues it is worth checking the DeepSkyStacker Wiki: <https://groups.io/g/DeepSkyStacker/wiki>

If you aren't a member the DeepSkyStacker mailing list, please join by sending an email to:

DeepSkyStacker+subscribe@groups.io

Once you have successfully joined the mailing list please report the problem on the mailing list by sending an email to:

DeepSkyStacker@groups.io

Your email should contain the following information:

**Describe the bug**
A clear and concise description of what the bug is.  Please specify the version of DeepSkyStacker and if relevant whether the problem is with DeepSkyStacker Live or DeepSkyStacker Command Line

**Provide necessary data to reproduce the problem**
Most problems with DeepSkyStacker will need you to upload a minimal set of files that allow the problem to be recreated including a dssfilelist to a file sharing site.  Obviously if the problem can be recreated with any (or no data) such as a UI problem then we don't need that.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by astrobob56** [comment]
I have been running 5.1.4 for the past six months without any issues. I upgraded to 5.1.6 today and attempted to process images of the Dark Shark Nebula I captured last night. The registration worked as expected. However, when I attempted to stack the frames, the application aborted and closed soon after computing offsets. There are a total of 68 frames I am attempting to stack.  I downgraded to 5.1.5 and the application registered and stacked the frames as expected. No crash.

I will continue to run 5.1.5 for now.
